www/source/blog/2017-10-27-distributed-java-application.html.md

```diff
@@ -200,7 +200,7 @@ So with our application freshly built, let's see if we can actually run it. As I
 
 ```studio
 [3][default:/src:0]# hab sup status
-mwrock/np-mongodb/3.2.9/20171013092907 (standalone), state:up, time:PT238.002019600S, pid:539, group:np-mongodb.default, style:transient
+mwrock/np-mongodb/3.2.9/20171013092907 (standalone), state:up, time:PT238.002019600S, pid:539, group:np-mongodb.default
```

Comment:
Is it worth changing old blog posts like this? The output format for `hab sup status` has changed since this was written, but we're not going to reformat it here.

Reply:
Eh, I update posts to try and keep things in line but as you point out I'm not going to rewrite this whole thing to be 100%. It was a find and replace and I just let it go through. I can revert it back